pkg/controller/install/caresources.go

```diff
@@ -0,0 +1,543 @@
+package install
```

Comment:
naming nit: the filename also reads as `care sources` - what about something like "certs"?

Reply:
Fair point - renamed to `certResources`.